tickit/adapters/interpreters/command/command_interpreter.py

```diff
@@ -12,16 +20,17 @@ class Command(Protocol):
 
     #: A flag which indicates whether calling of the method should trigger an interrupt
     interrupt: bool
+    # format: Optional[str] = None
 
     @abstractmethod
-    def parse(self, data: bytes) -> Optional[Sequence[AnyStr]]:
+    def parse(self, data: AnyStr) -> Optional[Tuple[Sequence[AnyStr], int, int, int]]:
```

Comment:
Feels like `Tuple[Sequence[AnyStr], int, int, int]` should be given a `NewType` declaration or even a `dataclass`